cmd/kube-scheduler/app/options/options.go

```diff
@@ -186,11 +184,18 @@ func (o *Options) ApplyTo(c *schedulerappconfig.Config) error {
 			return err
 		}
 
-		// use the loaded config file only, with the exception of --address and --port. This means that
-		// none of the deprecated flags in o.Deprecated are taken into consideration. This is the old
-		// behaviour of the flags we have to keep.
+		// use the loaded config file only, with the exception of --address and --port.
```

Comment:
This comment refers to `o.CombinedInsecureServing.ApplyToFrmLoadedConfig` below

Reply:
moved